flexmeasures/auth/policy.py

```diff
@@ -11,10 +11,12 @@
 from werkzeug.exceptions import Unauthorized, Forbidden
 
 
-PERMISSIONS = ["create-children", "read", "read-children", "update", "delete"]
+PERMISSIONS = ["create-children", "read", "update", "delete"]
```

Comment:
This role was introduced years ago (according to git blame) and was not used anywhere.